packages/edit-site/src/components/page-pages/index.js

```diff
@@ -70,24 +70,32 @@ export default function PagePages() {
 				...view,
 				filters: {
 					...view.filters,
-					status: defaultStatuses,
+					status: {
+						in: defaultStatuses,
+					},
 				},
 			} );
 		}
 	}, [ defaultStatuses ] );
 
-	const queryArgs = useMemo(
-		() => ( {
+	const queryArgs = useMemo( () => {
+		const filters = {};
+		if ( view.filters?.status?.in ) {
```

Comment:
I feel like we should have this info in the filters structure and not manually handled here.

Maybe something like:
```
author: {
      in: { $key: 1, }
      notIn: { $key: 2},
    },
```

I haven't thought about it depth, but it seems we need a consistent way to declare different types of filters(like here with `in, notIn`) and something to construct the query. Similar thing might be needed for sorting, searching etc..

Reply:
which "filters" structure you're talking about? I think constructing the query should be decoupled from the structure we use within the data views. We could have some config to help us with this but I see this as a specific function "viewToPostQuery" a "viewToTemplateQuery" might be different (this is a function that is specific to the REST API being used or any kind of query)

Reply:
Yep, right.